src/executables/util_cs.c

```diff
@@ -3192,13 +3198,35 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
   if (master_node_name != NULL)
     {
       check_master_info = true;
+      ha_node_list_p = prm_get_string_value (PRM_ID_HA_NODE_LIST);
+      start_node = (char *) strchr (ha_node_list_p, '@');
+      if (start_node == NULL || ha_node_list_p == start_node)
+	{
+	  return NULL;
+	}
+      ha_node_list_pp = util_split_string (start_node + 1, " ,:");
     }
 
   check_applied_info = utility_get_option_bool_value (arg_map, APPLYINFO_APPLIED_INFO_S);
   log_path = utility_get_option_string_value (arg_map, APPLYINFO_COPIED_LOG_PATH_S, 0);
   if (log_path != NULL)
     {
-      log_path = realpath (log_path, log_path_buf);
+      if (realpath (log_path, log_path_buf) != NULL)
+	{
+	  log_path = log_path_buf;
+	}
+      memset (active_log_path, 0, PATH_MAX);
+      fileio_make_log_active_name ((char *) active_log_path, log_path, database_name);
+      if (!fileio_is_volume_exist ((const char *) active_log_path))
+	{
+	  er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_MOUNT_FAIL, 1, active_log_path);
+	  error = ER_LOG_MOUNT_FAIL;
+	}
+      if (error != NO_ERROR)
+	{
+	  printf ("\n%s\n\n", db_error_string (3));
+	  goto print_applyinfo_usage;
+	}
```

Comment:
It seems to be merged.
```suggestion
      if (!fileio_is_volume_exist ((const char *) active_log_path))
	{
	  er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_MOUNT_FAIL, 1, active_log_path);
	  error = ER_LOG_MOUNT_FAIL;
	  printf ("\n%s\n\n", db_error_string (3));
	  goto print_applyinfo_usage;
	}
```

Reply:
That's right. It will be better.